### PR TITLE
Restructure of docs completed

### PR DESCRIPTION
**Pull request by BlackDragonBE**
I have created and renamed the categories as we discussed. I've also added placeholder topics.

### CODE REVIEW

**Comment by BlackDragonBE** [comment]
@rcannood 
This PR is ready for review now.
I've made the following extra changes:

- Changed "viash" to "Viash" except for commands
- Added level 2 title: Advanced Topics
- Added level 2 title: Managing Viash Projects
- Moved namespaces explanation to Namespaces page
- Moved Dynamic Config Modding page to Advanced Topics
- Hide empty topics (I used the draft attribute)
- Replaced code of conduct email with info@viash.io 
- Adjusted "Contributing Guidelines": Less enthusiasm for IntelliJ IDEA, replaced happywithgitr link, moved code contribution section below other guidelines 
- Ran a global spell check and fixed small typos in text and code comments
- Add draft of Scripting Variables under Advanced Topics

File: content/docs/running/executables-docker.md [hunk 1]
```diff
@@ -15,29 +15,59 @@ toc: true
...
+### ifneedbepullelsecachedbuild
+
+if the image does not exist locally, pull the image else build it with
+caching
+
+### donothing / meh
+
+do not build or pull anything
+
+### build / b
```
Comment (rcannood) [inline content/docs/running/executables-docker.md:65]: Move `/ b` to `alwaysbuild / build`. Same for  `/ cb``

`pull/p` means viash will try to pull the container from docker hub or some other docker registry.

`push` will push the container to docker hub or some other docker registry. 

`pushifnotpresent` will push the container to docker hub or some other docker registry if that tag does not exist yet.

**Review by rcannood — APPROVED** [review]
Looks very good. I like the reordering of the content, but more importantly I like the examples for the different commands and the contributing guidelines! :=)

File: content/docs/running/executables-docker.md (hunk 1 above)
Comment (BlackDragonBE) [reply content/docs/running/executables-docker.md:65]: @rcannood Got it! Thanks for the clarification, I'll make the needed adjustments.

**Pull request merged**